ilc/server/logger/fastify.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const _ = require('lodash');
+const pluginManager = require('../pluginManager/factory');
+
+
+let fastifyConf = pluginManager.getReportingPlugin();
+if (fastifyConf === null) {
+    const hyperid = require('hyperid')
+    const instance = hyperid({urlSafe: true, fixedLength: true});
+
+    fastifyConf = {
+        logger: require('./index'),
+        genReqId: instance,
+    }
+}
+
+module.exports = _.omit(_.pick(fastifyConf, ['logger', 'requestIdLogLabel', 'genReqId']), _.isEmpty);
```

Comment:
Where does requestIdLogLabel exist?
Should we add the default one?

Reply:
See https://www.fastify.io/docs/v2.13.x/Server/#requestidloglabel